pages.es/osx/wacaw

```diff
@@ -0,0 +1,24 @@
+# wacaw
+
+> Una pequeña herramienta de línea de comandos para macOS que le permite capturar imágenes fijas y videos desde una cámara adjunta.
```

Comment:
I propose a chance to the English page in this PR because the description was a little subjective.
```suggestion
> Herramienta de línea de comandos para macOS para capturar imágenes fijas y videos desde una cámara adjunta.
```